doc/usage/tutorial.md

```diff
@@ -45,10 +33,68 @@ NAME        	REVISION	UPDATED                 	STATUS  	CHART                	NA
 dozing-otter	1       	Thu Jun 14 18:52:15 2018	DEPLOYED	kube-batch-0.4.1    	kube-system
 ```
 
-NOTE: `kube-batch` need to collect cluster information(such as Pod, Node, CRD, etc) for scheduling, so the service account used by the deployment must have permission to access those cluster resources, otherwise, `kube-batch` will fail to startup. For users who are not familiar with Kubernetes RBAC, please copy the example/role.yaml into `$GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch/templates/` and reinstall batch.
+## Customize kube-batch
+
+Please check following configurable fields and customize kube-batch based on your needs.
+
+### kube-batch configuraions
```

Comment:
Add customization section for
- kube-batch arguments
- scheduler-conf configuration